.evergreen/shared.yml

```diff
@@ -89,14 +89,17 @@ functions:
           MONGOSH_DECOMPRESS: ${decompress}
         working_dir: ui/evergreen
         command: make get-mongosh
-    - command: subprocess.exec
+    - command: shell.exec
       type: setup
       params:
         background: true
         # Turn off silent to enable DB logging for debugging purposes
         silent: true
         working_dir: ui/evergreen
-        command: make start-mongod
+        shell: bash
+        script: | 
+          mkdir -p bin
+          make start-mongod > bin/mongod-logs.txt
```

Comment:
Did you try the [`--logpath`](https://www.mongodb.com/docs/manual/reference/program/mongod/#std-option-mongod.--logpath) option? The native way may be preferable (I think as this is we'd omit stderr from the file)

Reply:
I included stderr in the log file with 2>&1. This will stream stderr to stdout